src/compiler/rust.rs

```diff
@@ -2895,13 +2897,36 @@ c:/foo/bar.rs:
 
     #[cfg(feature = "dist-client")]
     #[test]
-    fn test_parse_rustc_z_ls() {
+    fn test_parse_rustc_z_ls_old() {
```

Comment:
I'd recommend using an absolute identifier to describe the difference in behaviour that you're testing here: "old" and "new" is relative, and will mean something different when these tests are investigated in, say, 2 years time.

Perhaps something like `test_parse_rustc_ls_pre_1_60()` (or, whichever rust version changed the behaviour).

Reply:
I'm not sure in which version it changed, from what I tested at least 1.58. Maybe I can find it within `rustc` source ...

Reply:
https://github.com/rust-lang/rust/commit/cef3ab75b12155e0582dd8b7710b7b901215fdd6